examples/directml/llama_v2/config_llama_v2.json

```diff
@@ -13,207 +13,207 @@
+                                "past_key_values_out.8.key",
+                                "past_key_values_out.8.value",
+                                "past_key_values_out.9.key",
+                                "past_key_values_out.9.value",
+                                "past_key_values_out.10.key",
+                                "past_key_values_out.10.value",
+                                "past_key_values_out.11.key",
+                                "past_key_values_out.11.value",
+                                "past_key_values_out.12.key",
+                                "past_key_values_out.12.value",
+                                "past_key_values_out.13.key",
+                                "past_key_values_out.13.value",
+                                "past_key_values_out.14.key",
+                                "past_key_values_out.14.value",
+                                "past_key_values_out.15.key",
+                                "past_key_values_out.15.value",
+                                "past_key_values_out.16.key",
+                                "past_key_values_out.16.value",
+                                "past_key_values_out.17.key",
+                                "past_key_values_out.17.value",
+                                "past_key_values_out.18.key",
+                                "past_key_values_out.18.value",
+                                "past_key_values_out.19.key",
+                                "past_key_values_out.19.value",
+                                "past_key_values_out.20.key",
+                                "past_key_values_out.20.value",
+                                "past_key_values_out.21.key",
+                                "past_key_values_out.21.value",
+                                "past_key_values_out.22.key",
+                                "past_key_values_out.22.value",
+                                "past_key_values_out.23.key",
+                                "past_key_values_out.23.value",
+                                "past_key_values_out.24.key",
+                                "past_key_values_out.24.value",
+                                "past_key_values_out.25.key",
+                                "past_key_values_out.25.value",
+                                "past_key_values_out.26.key",
+                                "past_key_values_out.26.value",
+                                "past_key_values_out.27.key",
+                                "past_key_values_out.27.value",
+                                "past_key_values_out.28.key",
+                                "past_key_values_out.28.value",
+                                "past_key_values_out.29.key",
+                                "past_key_values_out.29.value",
+                                "past_key_values_out.30.key",
+                                "past_key_values_out.30.value",
```

Comment:
nit: for the output, we use present.[index].key/value in other examples.

Reply:
For example, https://github.com/huggingface/transformers/blob/651408a077f842e76e75bfc7d02b8ac38eeb6480/src/transformers/models/bart/configuration_bart.py#L240

do we need unify them?